src/api/base.ts

```diff
@@ -27,7 +26,7 @@ export type BaseOptions = {
   secureExpire?: string
   store?: boolean
 
-  cancel?: CancelController
+  signal?: AbortSignal
```

Comment:
Shouldn't it be `abortSignal`? In case there's a new spec for `SomeAnotherController` with signals.

Reply:
`fetch` has the same API, so i think consistent solution would be better
 
```js
let controller = new AbortController();
fetch(url, {
  signal: controller.signal
});
```

https://javascript.info/fetch-abort